config_repo/config.sh.repo

```diff
@@ -1,139 +1,187 @@
 #!/bin/bash
 
-# Choose between ZWO or RPiHQ or auto (prefers ZWO if present)
-CAMERA="auto"
+# NOTES:
+#  - Variable names must start at the beginning of a line.
+#  - There should be nothing after a variable's value, i.e., no comments on the same line.
+#  - Values can optionally be quoted with double quotes, NOT single quotes.
+#  - If a variable occurs multiple times, only the last value is used.
+
+# Choose between "ZWO", "RPiHQ", or "auto".  "auto" prefers ZWO if present.
+CAMERA=""
+
+if [ "${CAMERA}" = "" ]; then
+	echo "${RED}ERROR: Please set CAMERA in config.sh${NC}"
+	exit 1
+fi
+
+
+########## Images
+# Set to "true" to upload the current image to your website.
+IMG_UPLOAD="false"
+UPLOAD_IMG="${IMG_UPLOAD}"	# !!! For backwards compatibility - do not change !!!
```

Comment:
this too. I'm thinking of a  "No User-Serviceable Parts" section. "Touch any of this stuff and your allsky will die."